### PR TITLE
Fix: Fix jumping behavior when hovering over tabs of open report/alert.

### PR DESCRIPTION
**Pull request by a-h-abdelsalam**
## What
Prefix pseudo classes with &.

## Why

Pseudo selectors no longer work when not prefixed with & in styled-components 6 which causes jumping behavior when hovering over tabs in open reports or alerts. See https://github.com/styled-components/styled-components/issues/4177

## References
GEA-382
Closes: https://github.com/greenbone/gsa/issues/3914

### CODE REVIEW

**Comment by cfi-gb** [comment]
You could also add something like e.g. the following to the initial issue description and the issue would be then automatically closed once this PR is merged:

Closes: #3914

File: src/web/components/tab/tab.js [hunk 1]
```diff
@@ -47,13 +47,13 @@ const StyledDiv = styledExcludeProps(styled.div, ['active'])`
   margin-bottom: ${props => (props.active ? '-2px' : undefined)};
   border-top: ${props =>
     props.active ? '2px solid ' + Theme.green : '2px solid ' + Theme.white};
-  :hover {
+  &:hover {
     border-top: ${props =>
       props.active
-        ? '2px solid ' + Theme.white
+        ? '2px solid ' + Theme.green
```
Comment (bjoernricks) [inline src/web/components/tab/tab.js:53]: Is this change intended?

Comment (a-h-abdelsalam) [reply src/web/components/tab/tab.js:53]: Yes in order to keep the top border green when a tab is highlighted.

**Review by timopollmeier — APPROVED** [review]
